app/src/main/java/waiter/RouteDirector.java

```diff
@@ -0,0 +1,54 @@
+package waiter;
+
+public class RouteDirector {
+
+    public Route constructGetSimpleGet(RouteBuilder routeBuilder) {
+        return routeBuilder.newUp()
+                .url("/simple_get")
+                .method("GET")
+                .status("200 OK")
+                .headers("Content-Length:")
+                .body("")
+                .build();
+    }
+
+    public Route constructHeadSimpleGet(RouteBuilder routeBuilder) {
+        return routeBuilder.newUp()
+                .url("/simple_get")
+                .method("HEAD")
+                .status("200 OK")
+                .headers("Content-Length:")
+                .body("")
```

Comment:
It's interesting to me that I need to define an empty body for this `HEAD` method.

If I were to specify a value in this body and make a `HEAD` request to `simple_get` would the response include a body?

Reply:
What about content type? I don't see any reference for that at the moment. Is it something we need to worry about?

Reply:
As of right now, I don't think the absence or presence of content type is something we need to worry about

Reply:
This is unrelated to this PR, but if we're returning a body, we should also specify the content type per the [HTTP specifications](https://www.rfc-editor.org/rfc/rfc7231#section-3.1.1.5). Given the number of comments already there and to avoid expanding the scope of this PR: I'm fine if we address this in a separate PR. Thoughts?